vulcan/lib/orchestration.rb

```diff
@@ -49,7 +49,10 @@ def load_json_inputs!(storage)
     # bad inputs.  Validating here would start to conflate potential storage or logic errors with simple user input
     # errors.
     def load_json_payload!(storage, reference)
-      if (payload = reference[:json_payload])
+      if (reference.key?(:json_payload))
```

Comment:
I'm actually a little curious what `false files` are?

Reply:
There are default boolean values that can be set to `false`. When that's the case, the old logic would evaluate `payload` as `false` and not generate a file with that value written to it...

Reply:
Which would then mean the orchestrator would never consider those steps as ready to run, because they were missing input files :smile:

Reply:
Is this regarding `regress_<cell-feature>` inputs?  If so, the files would be a string recording either "True" or "False" as UI-inputs are transferred from vulcan to archimedes as files.

Reply:
Is, that's how the issue showed up. Hm, so you might have to update the script to do some Python manipulation of the string value instead of just reading it in an casting it to a bool, because I'm pretty sure it will be recorded as `false` and `true` since Ruby uses lowercase...

Reply:
In which case, casting it to `bool` in Python might cast both values as `True`

Reply:
It did, and so I added an interpreter function that ignores casing and calls yes/true/y/1 -> True, else False. (We can update/trim that.)